fabric-common/lib/Endorser.js

```diff
@@ -59,7 +59,7 @@ class Endorser extends ServiceEndpoint {
 	 * @returns {Promise} A Promise for a Object that is the Protobuf
 	 *  'protos.ProposalResponse' see fabric-protos/protos/peer/proposal_response.proto
 	 */
-	sendProposal(signedProposal, timeout) {
+	async sendProposal(signedProposal, timeout) {
```

Comment:
Why do we want to put the "async" on this method ?

Reply:
Oops, I was going to use await internally but then didn't. I'll remove the async

Reply:
Done